configure.ac

```diff
@@ -4,17 +4,17 @@ AC_PREREQ([2.60])
 # the API. All changes in experimental modules are treated as
 # backwards-compatible and therefore at most increase the minor version.
 define(_PKG_VERSION_MAJOR, 0)
-define(_PKG_VERSION_MINOR, 6)
-define(_PKG_VERSION_PATCH, 1)
-define(_PKG_VERSION_IS_RELEASE, false)
+define(_PKG_VERSION_MINOR, 7)
+define(_PKG_VERSION_PATCH, 0)
+define(_PKG_VERSION_IS_RELEASE, true)
 
 # The library version is based on libtool versioning of the ABI. The set of
 # rules for updating the version can be found here:
 # https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
 # All changes in experimental modules are treated as if they don't affect the
 # interface and therefore only increase the revision.
-define(_LIB_VERSION_CURRENT, 5)
-define(_LIB_VERSION_REVISION, 1)
+define(_LIB_VERSION_CURRENT, 6)
+define(_LIB_VERSION_REVISION, 0)
 define(_LIB_VERSION_AGE, 0)
```

Comment:
To make it easier to verify versioning changes, I suggest including the output of the `./tools/check-abi.sh` tool in the PR description in the future.

For example, for this one:
<img width="938" height="783" alt="image" src="https://github.com/user-attachments/assets/36df78cc-a60c-412d-bcb4-451285be071b" />